elasticdl/python/master/task_dispatcher.py

```diff
@@ -134,6 +142,41 @@ def create_tasks(self, task_type, model_version=-1):
                 self._todo.extend(tasks)
         return tasks
 
+    def _create_save_model_task(self):
+        # For the SavedModel Task, we need a shard of data to build the model
+        shards = self._training_shards
+        assert shards is not None
+
+        (shard_name, (start_ind_this_shard, num_records_this_shard)) = next(
+            iter(shards.items())
+        )
+        start_ind_this_task = start_ind_this_shard
```

Comment:
No need to assign `start_ind_this_shard` to `start_ind_this_task`, just use it directly in _Task init.

Reply:
We also use this variable in 
end_ind_this_task = start_ind_this_shard + min(
            self._records_per_task, num_records_this_shard
        )
I prefer keeping it to make the code more readable.